transfer_learning.py

```diff
@@ -118,7 +118,9 @@ def train_model(self, train, test):
             'eval_steps': 100,
             'logging_steps': 100,
             'per_device_eval_batch_size': self.per_device_train_batch_size,
-            'seed': 0
+            'seed': 0,
```

Comment:
I like this idea in practice, but does that mean I would need to rerun my models to line up the training processes for the paper? Maybe we just add this as something we would add in future iterations of this analysis and remove for now?

Reply:
hmmm yeah I think i'm gonna take this out. It's a trick I was using occasionally locally but I don't think it's something we need as part of the official development process